website/pages/docs/plugins/sources/gcp/policies.md

```diff
@@ -8,7 +8,7 @@ This page documents the available CloudQuery SQL Policies for GCP. See the [read
 ### Requirements
 GCP CIS v1.2.0 requires the following tables to be synced before the policy is executed:
 
-```yaml copy
```

Comment:
I added this in https://github.com/cloudquery/cloudquery/pull/7303, but didn't realize it's auto generated. I'll update the policy script separately